drivers/net/wireless/marvell/mwifiex/pcie.c

```diff
@@ -284,6 +284,9 @@ static int mwifiex_pcie_probe(struct pci_dev *pdev,
 		return -1;
 	}
 
+	/* check quirks */
+	mwifiex_initialize_quirks(card);
```

Comment:
I think this should be done before calling `mwifiex_add_card()`, so that it's set before it can be accessed somewhere else.

Reply:
Moved `mwifiex_initialize_quirks()` before `mwifiex_add_card()` by the new rebase.